include/psa/crypto_values.h

```diff
@@ -634,7 +651,8 @@
  *         algorithm identifier.
  */
 #define PSA_ALG_IS_HASH(alg)                                            \
-    (((alg) & PSA_ALG_CATEGORY_MASK) == PSA_ALG_CATEGORY_HASH)
+    ((((alg) & PSA_ALG_CATEGORY_MASK) == PSA_ALG_CATEGORY_HASH) &&      \
+      !PSA_ALG_IS_POLICY(alg))
```

Comment:
Given the precedent that `PSA_ALG_IS_HASH` is true for a hash wildcard, I don't think this change is right. `PSA_ALG_IS_category` should remain true for wildcards.

Reply:
Fixed